src/main/java/crawlercommons/sitemaps/extension/VideoAttributes.java

```diff
@@ -424,17 +424,17 @@ public void setLive(Boolean live) {
         /**
          * Video price
          */
-        private float price;
+        private Float price;
 
-        public VideoPrice(final String currency, final float price) {
```

Comment:
Not a comment on your change, but this use of `final` in method parameters is...not very useful or standard :)